.envrc

```diff
@@ -1,3 +1,4 @@
+export RMW_IMPLEMENTATION=rmw_fastrtps_cpp
```

Comment:
this should go in the flake I thing

Reply:
Ah yest now that I can set some variables in the shell I agree.